v3/profile.d/klam.sh

```diff
@@ -0,0 +1,2 @@
+KLAM_USER=$(who -m | awk '{print $1}')
+PROMPT_COMMAND='RETRN_VAL=$?;logger -p local6.debug "$KLAM_USER [$$]: $(history 1 | sed "s/^[ ]*[0-9]\+[ ]*//") [$RETRN_VAL]"'
```

Comment:
Got rid of this and made a change to the global command prompt instead to use the `who -m` voodoo instead.  This wasn't working correctly under CoreOS as it was, so away it goes!